internify/src/components/pages/styles/Login.css

```diff
@@ -1,9 +1,9 @@
-.login {
+/* .login {
```

Comment:
if this is not needed anymore, feel free to just remove altogether. We want to refrain from adding as many comments as possible